src/modules/core/components/BannedTag/BannedTag.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { HTMLAttributes } from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import { SimpleMessageValues } from '~types/index';
+import Icon from '~core/Icon';
+
+import styles from './BannedTag.css';
+
+const displayName = 'BannedTag';
+
+interface Props extends HTMLAttributes<HTMLSpanElement> {
+  text?: MessageDescriptor | string;
+  textValues?: SimpleMessageValues };
```

Comment:
You don't need the ending curly brace here - that's why there is a linting error that github is showin
<img width="677" alt="Screenshot 2021-11-17 at 18 05 01" src="https://user-images.githubusercontent.com/34057551/142247441-b4c91a26-3740-4d6d-a7fe-d66b450e4ab4.png">
g

Reply:
Thanks for the feedback, should look good now please re-review thx! gn